packages/inventory/src/constants.ts

```diff
@@ -11,7 +11,7 @@ export const OPENSRP_ENDPOINT_LOCATION = 'location';
 export const OPENSRP_PRODUCT_CATALOGUE = 'product-catalogue';
 
 // router routes
-export const INVENTORY_SERVICE_POINT_LIST_VIEW = '/inventory/list';
+export const INVENTORY_SERVICE_POINT_LIST_VIEW = '/inventory';
```

Comment:
what necessiated this change, I thought the whole point of this was so that we could decouple the routes from the app, such that they can look however they like

Reply:
this Pr does that as well as it had this functionality (when on a nested route not present in sidebar the the parent or related route should be active (#504 #238))

I.e when we are on `/inventory/add' page in the sidebar it will related it self to  route closely matching to itself which will be `/inventory` it cannot relate it self to `/inventory/list` because `/inventory/list` can be a different one and we are not sure like we also have `/inventory/edit` and we cannot relate `/inventory/add` to that key

Reply:
@ashfahan please say that in another way, did not understand what you meant